src/fides/api/schemas/privacy_request.py

```diff
@@ -339,3 +340,42 @@ class RequestTaskCallbackRequest(FidesSchema):
     rows_masked: Optional[int] = Field(
         default=None, description="Number of records masked, as an integer"
     )
+
+
+class PrivacyRequestFilter(FidesSchema):
```

Comment:
Should we consider forbidding extra fields to be passed in here? they won't have an effect, given how you're explicitly defining them by name when calling `_shared_privacy_request_search`, I was just surprised that I could pass in the old `identity` value in the request body with the new endpoint (which is just ignored)

Reply:
Sounds good, less surprises that way